### ISSUE TITLE
Probleme double tap iOS with VoiceOver

### ISSUE THREAD

**Issue by math65** [BearWare/TeamTalk5#604]
When VoiceOver is on if you double tap with two fingers, the microphone turns off but you cannot turn it back on with the double tap.

**Comment by bear101** [comment]
If you uncheck "Push to Talk lock" in Preferences then you can simply double tab on TX to enable/disable transmission.

Note that if you have voice activation enabled at the same time you'll see some weird behavior.

**Comment by bear101** [comment]
No response. Closing...
